app/assets/javascripts/cascade/level/career_blocks.js

```diff
@@ -0,0 +1,202 @@
+$(function () {
+  if ($(".career-block-widget__container").children().length > 1) {
+    $(".career-block-widget__container").addClass(
+      "career-block-widget__container--3-or-more-items"
+    );
+  }
+
+  $.each($(".career-block-widget"), function (ind) {
+    $(this).attr("id", "career-block-widget__" + parseInt(ind + 1));
+    var id = $(this).attr("id");
+    // console.log("current id: " + id);
+    // console.log("this " + id);
+    var careerCode = $(this)
+      .find(".career-block-widget__career-code")
+      .text()
+      .replace(/\s/g, "");
+    console.log("careerCode " + careerCode);
+    var URL =
+      "https://services.onetcenter.org/ws/online/occupations/" +
+      careerCode +
+      "?client=chapman";
+    // console.log("url: " + URL);

```

Comment:
console.logs @nicholasnadel

Reply:
feel like these could be helpful to bedbug if need be. rather stick to debug @cbryant24 ?

Reply:
lol debug

Reply:
Sounds good wasn't sure if intentional or not

Reply:
fine with removing them -- up to you all

Reply:
@corejh